src/main/java/spark/embeddedserver/jetty/SocketConnectorFactory.java

```diff
@@ -113,6 +113,8 @@ private static HttpConnectionFactory createHttpConnectionFactory(boolean trustFo
         httpConfig.setSecureScheme("https");
         if(trustForwardHeaders)
             httpConfig.addCustomizer(new ForwardedRequestCustomizer());
+        httpConfig.setRequestHeaderSize(8192 * 2);
```

Comment:
This should be configurable from `Spark` similar to how `trustForwardHeaders` is handled. 

By doing so we'd keep the default at `8*1024` to keep backwards compatibility but allow customization per application should they choose to.